Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/OpenGeoMetadata/GeoCombine.git
-  revision: 77fde5526c0d1c4d9b6c79d58cd0acb7351ce46b
+  revision: b2688d4a75696e2211ae38f98989fe1a1c06542c
```

Comment:
GeoCombine v0.5.0 has just been :shipit:

Reply:
Okay, do you want us to get on a released version?

Reply:
Yeah we might as well